From ebe16ee2ae0e2e20e2950e5ceae179b98f1e4264 Mon Sep 17 00:00:00 2001
From: Petr Vorel <petr.vorel@gmail.com>
Date: Wed, 26 Jan 2022 23:20:57 +0100
Subject: sched.h: Add SCHED_DEADLINE

Defined in kernel v3.14, commit
aab03e05e8f7 ("sched/deadline: Add SCHED_DEADLINE structures & implementation")

Signed-off-by: Petr Vorel <petr.vorel@gmail.com>
---
 libc/sysdeps/linux/common/bits/sched.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/libc/sysdeps/linux/common/bits/sched.h b/libc/sysdeps/linux/common/bits/sched.h
index 9d05314f5..18a9fb98d 100644
--- a/libc/sysdeps/linux/common/bits/sched.h
+++ b/libc/sysdeps/linux/common/bits/sched.h
@@ -31,6 +31,7 @@
 #ifdef __USE_GNU
 # define SCHED_BATCH		3
 # define SCHED_IDLE		5
+# define SCHED_DEADLINE	6
 
 # define SCHED_RESET_ON_FORK	0x40000000
 #endif
-- 
cgit v1.2.3

